openshift/inspectJobWithCPU-template.yaml

```diff
@@ -42,6 +42,27 @@ objects:
             mark: cleanup
         spec:
           automountServiceAccountToken: false
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                  - matchExpressions:
+                      - key: CPU_FAMILY
+                        operator: In
+                        values:
+                          - ${CPU_FAMILY}
+                      - key: CPU_MODEL
+                        operator: In
+                        values:
+                          - ${CPU_MODEL}
+                      - key: PHYSICAL_CPUS
+                        operator: In
+                        values:
+                          - ${PHYSICAL_CPUS}
+                      - key: processor
+                        operator: In
+                        values:
+                          - ${PROCESSOR}
```

Comment:
Should we have `failureThreshold:` set to 0? In this way all the workflow will fail, we do not want the workflow to continue if there was an error before. At least not an error in the inspection run.

Reply:
To which `failureThreshold` are you referring to?

Reply:
it is few lines below sorry.

```
              livenessProbe:
                failureThreshold: 1
                initialDelaySeconds: 3600
                periodSeconds: 10
                tcpSocket:
                  port: 80
```

Reply:
Minimum value here is `1`, it cannot be set to `0`. This configuration is set to prevent pods to be stuck for too long - they can run at most 1 hour. But I wonder if that's enough for pyperformance.

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
That was the other option I had in mind, I think we need to increase it a bit, it might take a more than 1 hour to finish. Thanks :)

Reply:
I'll increase the delay to 2 hours and we can decide in the future whether it is okay to have it like that or potentially parametrize it.